Shopist/Shopist/Scenes/DetailViewController.swift

```diff
@@ -0,0 +1,45 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import UIKit
+
+final class DetailViewController: ListViewController {
```

Comment:
How about naming it `ProductsListViewController`? "Detail" doesn't describe what this screen is made for.